src/Symfony/Component/Notifier/Bridge/GoIP/GoIPTransportFactory.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Notifier\Bridge\GoIP;
+
+use Symfony\Component\Notifier\Exception\InvalidArgumentException;
+use Symfony\Component\Notifier\Exception\UnsupportedSchemeException;
+use Symfony\Component\Notifier\Transport\AbstractTransportFactory;
+use Symfony\Component\Notifier\Transport\Dsn;
+
+/**
+ * @author Ahmed Ghanem <ahmedghanem7361@gmail.com>
+ */
+final class GoIPTransportFactory extends AbstractTransportFactory
+{
+    private const SCHEME_NAME = 'goip';
```

Comment:
```suggestion

```

Reply:
I think the const usage is just fine ? Instead of maintaining the word in different places ?

There's also some bridges that follow the same pattern: [Novu](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Notifier/Bridge/Novu/NovuTransportFactory.php), [LineNotify](https://github.com/symfony/symfony/blob/6.4/src/Symfony/Component/Notifier/Bridge/LineNotify/LineNotifyTransportFactory.php)

Reply:
Ok 👌🏻